test/01_contraction/contraction_test.cpp

```diff
@@ -257,14 +257,17 @@ namespace hiptensor
             auto resource = getResource();
             resource->resizeStorage(lengths, elementBytes);
 
+            uint32_t seed = static_cast<uint32_t>(std::time(nullptr));
```

Comment:
We cannot reproduce the input data if seed is changing every time the test runs. Then we cannot reproduce a bug if there is. 

So we can set a fix number as seed.